tests/unit_tests/steps/test_root_inc_files.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+from unittest import mock
+
+import pytest
+
+from fab.constants import BUILD_OUTPUT
+from fab.steps.root_inc_files import RootIncFiles
+
+
+@pytest.fixture
+def common_data():
+    source_root = Path('/foo/source')
+    build_output = source_root.parent / BUILD_OUTPUT
+    step = RootIncFiles(source_root=source_root)
```

Comment:
Maybe make it clearer that `step` is the unit under test. Perhaps `test_unit`.

From a coding point of view I can see the attraction of having the fixture create your test unit as it's the same each time but for clarity it might be better to do it in the actual test.

Reply:
Agreed. It's now obvious what it is since we've pulled the constructor into the test so we've left the name. Happy to change that too if required.